### PR TITLE
include npm links

### PR DESCRIPTION
**Pull request by jgravois**
just a few copyedits to:

* clarify that both bower _and npm_ are options for installing the API
* add links to rene's blog which explains what is necessary to be successful with npm

also, do y'all think its time to make `4master` the default branch for the repo yet?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,34 @@
 # arcgis-js-api
 
-A minified, unbuilt version of the [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript/) to be used with [bower](http://bower.io).
+A minified, unbuilt version of the [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript/) to be used with bower and npm.
```
Comment (bsvensson) [inline README.md]: Could we keep the bower link even if we add npm?

Comment (jgravois) [reply README.md]: in my first draft i had links for both and looking at the markdown preview it just seemed distracting (especially given that the same links are repeated immediately below in `Features`).

that said, i'm happy to add it back (and make any other tweaks you like).

Comment (jgravois) [reply README.md]: done.

**Comment by bsvensson** [comment]
Thanks @jgravois

**Pull request merged**